operator/validator/task_executor.go

```diff
@@ -24,7 +24,7 @@ func (c *controller) taskLogger(taskName string, fields ...zap.Field) *zap.Logge
 func (c *controller) StopValidator(pubKey spectypes.ValidatorPK) error {
 	logger := c.taskLogger("StopValidator", fields.PubKey(pubKey[:]))
 
-	c.metrics.ValidatorRemoved(pubKey[:])
+	validatorsRemovedCounter.Add(c.ctx, 1)
```

Comment:
this isn't necessarily removed, but rather stopped (it could also be because of contract liquidation)